hazelcast/src/hazelcast/client/spi.cpp

```diff
@@ -1572,7 +1572,11 @@ ClientInvocation::invoke_on_selection()
         }
     } catch (exception::iexception&) {
         notify_exception(std::current_exception());
-    } catch (std::exception&) {
+    } catch (std::exception& exp) {
+        HZ_LOG(
+          logger_,
+          finest,
```

Comment:
This line is executed just before termination, so `finest` is not appropriate log type. Also, I don't think these lines are related with this bug. It can even be removed (if it is OK for you). But at least, it needs to be `severe` log type.

```suggestion
          severe,
```

Reply:
log is removed